docs/intro/development.md

```diff
@@ -0,0 +1,53 @@
+# Development getting started
+
+* Open in VS Code
+* Rebuild and Reopen in Container
+* Review the [`appsettings.json`][appsettings.json] and [sample data][data.csv] files
+* Set a breakpoint in [`src/Cli/Program.cs`][program.cs] and hit `F5` to run in debug mode
+* After the run completes, look in your local `src/Cli/sample` directory for the hashed output file
```

Comment:
Minor typo:
```diff
- `src/Cli/sample`
+ `src/Cli/samples`
```

Reply:
Thanks! Fixed in [e2db7ad](https://github.com/cal-itp/hashfields/pull/17/commits/e2db7ad630d36b39c9743b42dcc660299f676561)